src/ScottPlot/Plottable/Crosshair.cs

```diff
@@ -119,7 +129,7 @@ private void RenderHorizontalLine(PlotDimensions dims, Graphics gfx, Pen pen, Fo
             float yPixel = dims.GetPixelY(Y);
             gfx.DrawLine(pen, dims.DataOffsetX, yPixel, dims.DataOffsetX + dims.DataWidth, yPixel);
 
-            string yLabel = IsDateTimeY ? DateTime.FromOADate(Y).ToString(StringFormatY) : Y.ToString(StringFormatY);
+            string yLabel = ManualStringFormatterY != null ? ManualStringFormatterY(Y) : IsDateTimeY ? DateTime.FromOADate(Y).ToString(StringFormatY) : Y.ToString(StringFormatY);
```

Comment:
This ternary within a ternary is pretty unwieldly.

Reply:
Thank. I'll avoid using it in future.